elfi/__init__.py

```diff
@@ -1,5 +1,16 @@
 # -*- coding: utf-8 -*-
 
+# Solves apparent dependency on matplotlib for maximal compatibility
```

Comment:
Perhaps, but this seems least intrusive, as other modules don't have to worry about it.